### DISCUSSION TITLE
How to use with /api reverse proxy

### DISCUSSION THREAD

**Discussion by wug-ge** [nuxt/content#2305] [Q&A]
Hi!

My API get's reversed proxied from /api/* directly to my backend using a routeRules proxy in my nuxt.config.ts

Using Nuxt Content seems not to be possible for me, given it uses /api/_content to load the files which will end up at my backend.

Making an exception for "/api/_content/**" seems not to be possible, or is it?

I can't find an option for that there https://github.com/unjs/radix3#route-matcher

How would you solve this? What is the appropriate way?

Thanks!

**Comment by oneminch** [comment]
You can configure your content query route by configuring it using [`api`](https://content.nuxtjs.org/api/configuration#api). I'm not sure if that's what you were looking for.

```js
export default defineNuxtConfig({
  content: {
    api: {
      baseURL: '/my_content' // instead of the default '/api/_content'
    }
  }
})
```

You can read more about ways of configuring Nuxt Content [in the docs](https://content.nuxtjs.org/api/configuration)

**Comment by wug-ge** [comment]
@oneminch it is exactly what I was looking for, I must have missed it in the docs!

Thanks so much!
